app/src/main/java/com/example/marcinwisniewski/intivetrainingapp/utils/Constant.kt

```diff
@@ -0,0 +1,8 @@
+package com.example.marcinwisniewski.intivetrainingapp.utils
+
+class Constant {
+    companion object {
+        const val BASE_URL = "https://raw.githubusercontent.com"
+        const val ALL_MOVIES_URL = "/filippella/Sample-API-Files/master/json/movies-api.json"
```

Comment:
This API seems to be not really the API ;)

Reply:
Yes - i know that. I had some big problems with MOVIEDB API ( the key was proper, data from postman was also ok - i was still receiving null value list. To avoid waisting time on this i decided to use this sample row jsons - i remember we discussed it few days ago and we have a deal about it :)